surefire-providers/surefire-junit-platform/src/test/java/org/apache/maven/surefire/junitplatform/JUnitPlatformProviderTest.java

```diff
@@ -729,7 +730,8 @@ public void onlyJunitIncludeEngineIsDeclared()
         JUnitPlatformProvider provider = new JUnitPlatformProvider( providerParameters );
 
         assertThat( provider.getFilters() ).hasSize( 1 );
-        assertThat( provider.getFilters() ).allMatch( x -> x instanceof EngineFilter );
+        assertThat( Arrays.stream( provider.getFilters() ).collect( Collectors.toList()) )
```

Comment:
This is unnecessarily complicated.
I have checked it out in my code, and this works for me:
`assertThat( provider.getFilters() ).asList().element( 0 ).isInstanceOf( TestEngine.class )`
Does it work for you too?